app/va/va_profile/va_profile_client.py

```diff
@@ -363,14 +363,17 @@ def send_va_profile_notification_status(self, notification_data: dict) -> None:
             requests.Timeout: if the request to VA Profile times out
             RequestException: if something unexpected happens when sending the request
         """
+
         headers = {'Authorization': f'Bearer {self.va_profile_token}'}
         url = f'{self.va_profile_url}/contact-information-vanotify/notify/status'
 
         self.logger.debug(
-            'Sending notification status to VA Profile with url: %s | notification_id: %s | notification_status: %s',
+            'Sending notification status to VA Profile with url: %s | notification_id: %s '
+            '| notification_status: %s | service_name: %s',
```

Comment:
so like - should this be an info log instead?

Reply:
Proper functionality can be verified in Dev and Perf.  I manually verified by asking a Profile team member to check their logs.  We can do that in Prod if necessary.

Reply:
Sounds good to me!

Reply:
There is already an Info log on l397 with half of this data, so I think this does need to be an info log. Or add service name to the other log.

Reply:
I added it to the existing "info" log.